### DISCUSSION TITLE
Does MRTK support Varjo?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/MixedRealityToolkit-Unity#11410] [Q&A]
Our company is planning to buy some Varjo headsets (https://varjo.com/) to port our apps to. Does anyone know if MRTK supports these headsets? https://varjo.com/

**Comment by keveleigh** [comment]
It's been a few months, but we did do some MRTK3 testing on Varjo a while back. It did work, as far as I can remember! We were even able to get passthrough working using the [`Get/SetEnvironmentBlendMode` methods](https://docs.unity3d.com/Packages/com.unity.xr.openxr@1.6/api/UnityEngine.XR.OpenXR.Features.OpenXRFeature.html#UnityEngine_XR_OpenXR_Features_OpenXRFeature_SetEnvironmentBlendMode_UnityEngine_XR_OpenXR_NativeTypes_XrEnvironmentBlendMode_) (though we had to create a custom OpenXRFeature to call the protected methods).
